packages/system/src/Report/api/hooks.ts

```diff
@@ -40,8 +40,15 @@ export const useReports = (category?: ReportCategory) => {
     initialListParameters
   );
 
-  return useQuery(api.keys.paramList(queryParams), async () =>
-    api.get.list({ filterBy, sortBy, offset })
+  return useQuery(
+    api.keys.paramList(queryParams),
+    async () => api.get.list({ filterBy, sortBy, offset }),
+    {
+      onError: (e: Error) => {
```

Comment:
yeah. not ideal. Just wanted to be able to inform the user that they don't have permission

Reply:
Yes have to think about how to manage/define the report permissions...